meadow-client/package.json

```diff
@@ -27,6 +27,7 @@
     "node-sass": "^4.10.0",
     "popper.js": "^1.14.4",
     "pulp": "^12.3.0",
+    "purescript": "^0.11.7",
```

Comment:
I _think_ you've committed this by mistake. (It would be much better if we could use the yarn purescript instead of the nix one.)

Reply:
yeah a mistake, I don't think it works